GTG/gtk/data/context_menus.ui

```diff
@@ -141,7 +141,7 @@
       </item>
 
       <item>
-        <attribute name="label" translatable="yes">Pick a date...</attribute>
+        <attribute name="label" translatable="yes">Pick a Date...</attribute>
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en